cinema_service/settings.py

```diff
@@ -86,12 +87,15 @@
 
 DATABASES = {
     "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": BASE_DIR / "db.sqlite3",
+        "ENGINE": "django.db.backends.postgresql",
+        "NAME": os.environ.get("POSTGRES_DB"),
+        "USER": os.environ.get("POSTGRES_USER"),
+        "PASSWORD": os.environ.get("POSTGRES_PASSWORD"),
+        "HOST": os.environ.get("POSTGRES_HOST"),
+        "PORT": os.environ.get("POSTGRES_PASSWORD"),
```

Comment:
The 'PORT' setting for the database is incorrectly using the 'POSTGRES_PASSWORD' environment variable. It should be 'POSTGRES_PORT' instead.